layers.go

```diff
@@ -228,10 +267,11 @@ func (r *layerStore) Load() error {
 				}
 				names[name] = layers[n]
 			}
-			if pslice, ok := parents[layer.Parent]; ok {
-				parents[layer.Parent] = append(pslice, layers[n])
-			} else {
-				parents[layer.Parent] = []*Layer{layers[n]}
+			if layer.UncompressedDigest != "" {
```

Comment:
Nit of a nit.  Everywhere else in the code you reference compressed first, then uncompressed.  If/when you make changes here, could you flip the order of these if statements.  (I know, I know, it's my OCD kicking in).